models/salesforce_marketing_cloud__email_overview.sql

```diff
@@ -62,7 +62,7 @@ with emails as (
 
 ), joined as (
   select
-    emails.*,
+    emails.*, 
```

Comment:
Small note, but I don't think _all_ fields are necessary to be brought into the end model. For example, the first fields are `_fivetran_start`, `_fivetran_end`, and `_fivetran_active`. This makes reading the model at first glance a bit strange. Maybe move those to the end of the staging model or consider not bringing them through to the end model.